pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,6 +54,7 @@ type Response struct {
 
 type Client struct {
 	*client.Client
+	*generated.TestgroupClient
```

Comment:
This test client is only used in tests. Will this convince you it's ok to add the TestGroupClient here?